### ISSUE TITLE
Split keyboard option

### ISSUE THREAD

**Issue by ivanrancic** [maliit/keyboard#42]
It would enhance writing speed on tablets as it enables writing just with thumbs but it also does the same for the landscape mode on phones. 

![image](https://user-images.githubusercontent.com/84779952/119492621-92f4ed80-bd5f-11eb-9017-1ccb51e210b5.png)

**Comment by Cal-Nine** [comment]
Agreed. This combined with the ability to move the (split or unsplit keyboard) up and down the screen to avoid things you need to be visible would be very useful. 

#207
